caramel_launcher.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
```

Comment:
Move the launcher and ini to tests/  as well. pserve isn't production-recommended as it defaults to waitress as the server.